src/element.ts

```diff
@@ -141,12 +141,26 @@ export abstract class Element {
    */
   setStyle(style: ElementStyle | undefined): this {
     this.style = style;
+    const svg = this.getSVGElement();
+    if (svg) {
+      if (style?.fillStyle) {
+        svg.setAttribute('fill', style.fillStyle);
+        for (let i = 0; i < svg.children.length; i++) svg.children[i].setAttribute('fill', '');
```

Comment:
Can we add braces { } here even though it's a single line of code? I think always including braces is a simpler/safer coding style, and is easier to read.